### PR TITLE
Add 3DT token

### PR DESCRIPTION
**Pull request by Emilvooo**
Adding 3DAble token

### CODE REVIEW

File: tokens/test/0x5799543e3e0e0e89e5e917e2676d66b23013a654/info.json [hunk 1]
```diff
@@ -0,0 +1,6 @@
+{
+  "name": "3DAble Token",
+  "symbol": "3DT",
```
Comment (XJWX89) [inline tokens/test/0x5799543e3e0e0e89e5e917e2676d66b23013a654/info.json:3]: I have checked and found that the symbol provided does not match the information provided.

File: tokens/test/0x5799543e3e0e0e89e5e917e2676d66b23013a654/info.json [hunk 2]
```diff
@@ -0,0 +1,6 @@
+{
+  "name": "3DAble Token",
```
Comment (XJWX89) [inline tokens/test/0x5799543e3e0e0e89e5e917e2676d66b23013a654/info.json:2]: I have checked and found that the symbol provided does not match the information provided.  it should be `ThreeDAbleToken `

File: tokens/main/0x8fcddbb322b18d8bdaec9243e9f4c6eb8901e566/info.json [hunk 3]
```diff
@@ -0,0 +1,6 @@
+{
+  "name": "3DAble Token",
```
Comment (XJWX89) [inline tokens/main/0x8fcddbb322b18d8bdaec9243e9f4c6eb8901e566/info.json:2]: I have checked and found that the symbol provided does not match the information provided.  it should be `ThreeDAbleToken `

**Review by XJWX89 — CHANGES_REQUESTED** [review]
hi @Emilvooo ,I have found some issues in the data you provided. Please check and make the necessary changes. If you have any further questions, please feel free to ask

**Comment by Emilvooo** [comment]
@XJWX89 Thanks, changed it :)

**Comment by Emilvooo** [comment]
@libotony @XJWX89 could you check in on this? Thanks!

File: tokens/test/0x5799543e3e0e0e89e5e917e2676d66b23013a654/info.json [hunk 4]
```diff
@@ -0,0 +1,6 @@
+{
+  "name": "ThreeDAbleToken",
+  "symbol": "TDT",
```
Comment (libotony) [inline tokens/test/0x5799543e3e0e0e89e5e917e2676d66b23013a654/info.json:3]: The token symbol on testnet is `TDT`, as `3DT` on mainnet. But they share the same name and image, could you elaborate?

Comment (Emilvooo) [reply tokens/test/0x5799543e3e0e0e89e5e917e2676d66b23013a654/info.json:3]: The contract on mainnet was deployed later on, where we decided to change the symbol :)

Comment (libotony) [reply tokens/test/0x5799543e3e0e0e89e5e917e2676d66b23013a654/info.json:3]: Our recommendation is to keep the token symbol consistent between the mainnet and testnet. Please let us know if this aligns with your preferences.

**Comment by Emilvooo** [comment]
@libotony Changed testnet contract to be in line with mainnet. Let me know if this is correct now.

**Comment by libotony** [comment]
> @libotony Changed testnet contract to be in line with mainnet. Let me know if this is correct now.

Thanks, looking good now

**Review by libotony — APPROVED** [review]

**Review by XJWX89 — APPROVED** [review]

**Pull request merged**